### ISSUE TITLE
Browser blocking custom image to load

### ISSUE THREAD

**Issue by Dyzean** [Strencher/BdBrowser#4]
The browser seems blocking the external image from a themes to load
Tested on Opera GX and Chrome and other browser, the same issue happen

Console Log:
![image](https://user-images.githubusercontent.com/69681505/132126733-3e28c83f-2e7d-4088-86b3-76d9b4529a02.png)
![image](https://user-images.githubusercontent.com/69681505/132126813-5aaf8436-63c3-4312-9d41-3f088ea355eb.png)

**Comment by Strencher** [comment]
Yes, that's a CSP issue. I'll have to look into that and may have to regex all external url out from the code.

**Comment by Strencher** [comment]
Hmm, i loaded your theme and my browser isn't blocking it. Is that maybe a setting in chrome? I'm not sure tbh what's wrong.

**Comment by Dyzean** [comment]
weird, its blocking some other things from plugins tho for me, like fonts
![image](https://user-images.githubusercontent.com/69681505/132974459-b47e4463-10b9-471b-ab3d-878231a010ba.png)
